src/jit/lsraarmarch.cpp

```diff
@@ -403,7 +403,17 @@ void Lowering::TreeNodeInfoInitPutArgReg(
     info.srcCount++;
 
     // Set the register requirements for the node.
-    const regMaskTP argMask = genRegMask(argReg);
+    regMaskTP argMask = genRegMask(argReg);
```

Comment:
It is called from `Lowering::TreeNodeInfoInitCall(GenTreeCall* call)`. In `Lowering::TreeNodeInfoInitPutArgReg()`, not only it sets the PutArgReg node's candidates but also modifies the Call node's srcCount(`TreeNodeInfo& info` is the call node's info).